sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/complexTypesSuite.scala

```diff
@@ -453,46 +453,78 @@ class ComplexTypesSuite extends PlanTest with ExpressionEvalHelper {
     checkEvaluation(GetMapValue(mb0, Literal(Array[Byte](3, 4))), null)
   }
 
-  private val structAttr = 'struct1.struct('a.int)
```

Comment:
nit: `'struct1.struct('a.int.withNullability(false))..withNullability(false)`

Reply:
BTW seems like we only care about the nullability of the struct column, not the inner field. `'struct1.struct('a.int).withNullability(false)` should be good enough.

Reply:
I knew there was a better way to write this, thanks, done.